### DISCUSSION TITLE
Troyan detected (v1.5.6.0): Win32/Wacatac.B!ml

### DISCUSSION THREAD

**Discussion by AlbertoMassanet** [fedorovvl/tso_client#234] [General]
My Windows Defender has detected a troyan on last version. Anyone with that problem?

**Comment by fedorovvl** [comment]
u can check virustotal analysis report on release page

**Reply by AlbertoMassanet** [reply]
Yes.

https://i.imgur.com/GB9B5dZ.png

**Reply by fedorovvl** [reply]
yep, two noname antivirus detect smth..
